### DISCUSSION TITLE
Welcome to Gatesentry Discussions!

### DISCUSSION THREAD

**Discussion by fifthsegment** [fifthsegment/Gatesentry#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jlevitsk** [comment]
You've been working on this for a bit. What prompted the move to open source and are you 1 person or an org? Do you make other things? Just always so curious when I see something interesting to learn more about the origin story if you would be willing to share. I'm going to install this and tinker a bit. I work at a company that makes a Unified Endpoint Management product, but we don't have a proxy filter so we usually send people to the various commercial solutions like LightSpeed and I've always wondered if there were an opensource option and today I stumbled on yours and am so curious if there are certain sized organizations that could use it just fine.

**Comment by fifthsegment** [comment]
Thanks for your comment! I created this project as a hobby back in 2015-2016. It was initially designed to run on a Raspberry Pi, providing a filtered internet experience for families with young children. The setup included Squid3, DansGuardian, and a web-based GUI that I developed to manage the filters. The Raspberry Pi would broadcast a Wi-Fi signal, automatically proxying the traffic of connected devices.

In 2018, I took it a step further and wrote my own proxy server in Golang, though I hadn't open-sourced it until now. 

I'm driven by my passion for creating a safer online environment for young children, especially considering how early kids are getting their own internet-connected devices these days.

What I'm currently doing is a complete refactor of my code back from 2018, so it'll be 2-3 weeks before I get this into a stable release. (But you can definitely play with it now) . After that Im planning to test this and see how it compares to already existing solutions today, so I wouldnt recommend usage in a commercial setting yet  😅 

Thanks again for taking out the time to comment on my project 😃  (Apart from my full time job, this is the main thing that I currently work on)
